llvm/test/Transforms/LoopVectorize/AArch64/veclib-function-calls.ll

```diff
@@ -2128,25 +2276,98 @@ define void @copysign_f32(ptr noalias %in.ptr, ptr noalias %out.ptr) {
   ret void
 }
 
-declare double @fdim(double, double)
-declare float @fdimf(float, float)
+declare void @sincos(double, ptr, ptr)
+declare void @sincosf(float, ptr, ptr)
 
-define void @fdim_f64(ptr noalias %in.ptr, ptr noalias %out.ptr) {
-; SLEEF-NEON-LABEL: define void @fdim_f64
+define void @test_sincos(ptr noalias %a, ptr noalias %b, ptr noalias %c) {
```

Comment:
Similarly with `modf`, could also rename `test_sincos`,  `test_sincosf` to `sincos_f64`,  `sincos_f32`.

Reply:
I can change it as part of this patch.